src/components/editor/ArticleListController.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import List from "./List";
+
+var pageLength = 50;
+
+export default class ArticleListController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleUpdateSlugValue = this.handleUpdateSlugValue.bind(this);
+    this.safeSetState({
+      slugSearchValue: ""
+    })
+  }
+  static getFalcorPath(params) {
+    return [['articlesByTime', 'paginated', {from: pageLength*(params.page-1), length: pageLength}, ['title', 'slug']], ['articlesByTime', 'paginated', 'total']];
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    let pathStart = "/articles/page/" + this.props.params.page + "/article/slug/";
+    browserHistory.push(pathStart + this.state.slugSearchValue + "/");
+    this.safeSetState({
+      slugSearchValue: ""
+    });
+  }
+
+  handleUpdateSlugValue(event) {
+    this.safeSetState({
+      slugSearchValue: event.target.value
+    });
+  }
+
+  handlePageChange(increment, event) {
```

Comment:
I already addressed this issue quite comprehensively above, so refer to that mainly. But just wanted to ask if we could really use `<Link>` ? That's only clickable no? Because that was also my first thought that that was how I wanted to do it. Or do you mean you can like wrap a button in a `<Link>` ?